packages/mdc-notched-outline/README.md

```diff
@@ -82,6 +83,7 @@ Mixin | Description
 
 Method Signature | Description
 --- | ---
+`notch(activateNotch: boolean)` | Updates outline to activate/deactivate notch in outline path.
 `updateSvgPath(notchWidth: number, isRtl: boolean) => void` | Updates the SVG of the outline element with a notch calculated based off of the notchWidth. The notch will appear left justified, unless isRtl is true.
```

Comment:
Could we change this so there are two methods
`notch(notchWidth: number, isRtl: boolean) => void`
and
`closeNotch() => void`